src/unity/python/turicreate/toolkits/image_classifier/image_classifier.py

```diff
@@ -27,9 +27,23 @@
 from turicreate.toolkits._internal_utils import (_raise_error_if_not_sframe,
                                                  _numeric_param_check_range)
 
-def create(dataset, target, feature = None, model = 'resnet-50',
-           validation_set='auto', max_iterations = 10, verbose = True,
-           seed = None, batch_size=64):
+_DEFAULT_SOLVER_OPTIONS = {
+'convergence_threshold': 1e-2,
+'step_size': 1.0,
+'lbfgs_memory_level': 11,
+'max_iterations': 10}
+
+def create(dataset, target, features=None,
+    l2_penalty=0.01, l1_penalty=0.0,
+    solver='auto', feature_rescaling=True,
+    convergence_threshold = _DEFAULT_SOLVER_OPTIONS['convergence_threshold'],
```

Comment:
Currently, the solver can be several different algorithms (e.g. fista, newton, or lbfgs depending on regularization values, etc.), so let's put these in a **solver_params dictionary that can pass any number of keywords through to the solvers.